src/documents/SwitchAPI.md

```diff
@@ -0,0 +1,224 @@
+
+## Options ##
+
+### General options ###
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Values</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+            <td><code>selectors</code></td>
+            <td>JavaScript object defining CSS-style selectors for important DOM elements. See [Selectors](#selectors) for more information.</td>
+            <td>The object may contain the following keys:
+                <ul>
+                    <li><code>on</code></li>
+                    <li><code>off</code></li>
+                    <li><code>control</code></li>
+                </ul>
+            </td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    on: ".flc-switchUI-on",
+    off: ".flc-switchUI-off",
+    control: ".flc-switchUI-control"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+        <tr>
+            <td><code>strings</code></td>
+            <td>An object containing text to be used by the switch. These are displayed visually on the screen and/or provided as a label for an assistive technology.</td>
+            <td>The object may contain the following keys:
+                <ul>
+                    <li><code>label</code></li>
+                    <li><code>on</code></li>
+                    <li><code>off</code></li>
+                </ul>
+            </td>
+            <td>
+                <pre>
+                    <code>
+strings: {
+    // Specified by implementor
+    // text of label to apply the switch, must add to "aria-label" in the attrs block
+    label: "",
+    on: "on",
+    off: "off"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td><code>attrs</code></td>
+            <td>An object containing any HTML attributes to be added to the switch <code>control</code>.</td>
+            <td>
+                An object containing any key/value pairs representing valid HTML attributes and their respective values. In particular, an integrator should define a key/value pairing for <code>"aria-label"</code> or <code>"aria-labelledby"</code>.
+                <div class="infusion-docs-note"><strong>Note:</strong> If the switch controls another element on the page, <code>"aria-controls"</code> should be added to indicate the relationship.</div>
+            </td>
+            <td>
+                <pre>
+                    <code>
+attrs: {
+    role: "switch",
+    tabindex: 0
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td><code>model</code></td>
+            <td>The state of the switch.</td>
+            <td><code>true</code> is "on", <code>false</code> is off</td>
```

Comment:
Missing quotes for the word `off` to match `on` in the first half of the sentence. 

Missing period at the end of the sentence.